http/src/main/resources/init-resources/jupyter-docker-compose.yaml

```diff
@@ -21,7 +21,7 @@ services:
       - /usr/lib/hadoop-mapreduce:/usr/lib/hadoop-mapreduce
       - /usr/lib/hadoop-yarn:/usr/lib/hadoop-yarn
       - /usr/lib/hive:/usr/lib/hive
-      - /usr/lig/pig:/usr/lib/pig
```

Comment:
I thikn the executable is `/usr/lib/pig/bin`

```
root@saturn-29300fbd-7376-4b65-99c7-eb346f133c18-m:/usr/lib/pig/bin# pig --help
Apache Pig version 0.17.0 (r: unknown)
compiled Nov 30 2020, 11:16:03
```

Reply:
Hm, it looks like `/usr/bin/pig` is used..
```
root@saturn-62a6b6b2-11ab-45ed-82f2-ffde0ea5da0b-m:/usr/bin# which pig
/usr/bin/pig
```

Reply:
Will try this from the container